chapter09/mdw/Ex04.pl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+$^I= ".out";
+
+my $first = 1;
```

Comment:
I don't think this will work over multiple files, because this won't be reset when we open a new one.